lib/package-manager/install.js

```diff
@@ -27,17 +28,26 @@ function install(packageManager, context, next) {
     `Using temp directory: "${options.env['npm_config_tmp']}"`
   );
 
-  if (context.module.install) {
-    args = args.concat(context.module.install);
+  let nodeBin = 'node';
+  if (context.options.testPath) {
+    options.env.PATH = `${context.options.testPath}:${process.env.PATH}`;
+    nodeBin = which('node', {
+      path: options.env.PATH || process.env.PATH
```

Comment:
Can `options.env.PATH` ever not be `process.env.PATH` outside of this `if`? (i.e. does the `nodeBin` calculation need to occur outside of the `if` statement?)